server/src/entity/handover.ts

```diff
@@ -0,0 +1,36 @@
+import {
+    Entity,
+    PrimaryGeneratedColumn,
+    Column,
+    CreateDateColumn,
+    UpdateDateColumn,
+    DeleteDateColumn,
+    OneToOne,
+    JoinColumn,
+    BaseEntity,
+} from "typeorm";
+import { User } from "./user";
```

Comment:
these lines also need to modify.

Reply:
Completed with the update, and also updated other files.

Reply:
not seen any chaged.